src/main/kotlin/no/brreg/informasjonsforvaltning/organizationcatalogue/jena/JenaUtils.kt

```diff
@@ -81,6 +82,15 @@ private fun Resource.addPreferredNames(preferredNames: PrefLabel?): Resource {
     return this
 }
 
+private fun Resource.addOrgStatus(orgStatus: PrefLabel?): Resource =
+    if (orgStatus == null) this
+    else addProperty(ROV.orgStatus,
+        model.createResource(SKOS.Concept)
+            .addProperty(SKOS.prefLabel, orgStatus.nb, "nb")
+            .addProperty(SKOS.prefLabel, orgStatus.nn, "nn")
+            .addProperty(SKOS.prefLabel, orgStatus.en, "en")
```

Comment:
Organisasjonsstatus bør helst være enum. Jeg klarte ikke finne noen gode eksempler på hva det kan/bør være, men vi kan late som det er en enum med to verdier: NORMAL og LIQUIDATED.

Jeg ser to alternativer til en blank node, som kan være mer intuitive:
- dikte opp en URI som inneholder status og stoppe der
- hvis ikke 1), så lage globale SKOS.Concept ressurser i modellen og bruke URIer til disse ressurser på rov:orgStatus

Se følgende lenker for mer info:
https://www.w3.org/TR/vocab-regorg/#ref_rov_orgStatus

Reply:
Samesamesame for min del. 
@stigbd Dette er ditt bord. Kan fint gå over til Concept som er definert eksternt, men da må du bestemme deg for hvilke vi skal bruke.

Reply:
Det fins dessverre ingen publiserte begreper for denne statusen. Eg har kontakta Brønnøysundregistrene som har svart at dei har nokre upubliserte begreper, men det hjelper ikkje oss. Så inn til dei har publisert, synes eg at denne beste løysinga er å lage blanke noder for dei to statusane vi har laga her.

Ref mitt forslag i issuet eg lagde:
https://github.com/Informasjonsforvaltning/organization-catalogue/issues/84#issuecomment-773889473

Håpar dette var oppklarande.

Reply:
b9dbfd3

Er det lov?

Reply:
Ja, synes det ser fornuftig ut. Skrivebords-test: Korleis ville det ha sett ut viss Brønnøysundregistrene hadde publisert dette i begrepssamlinga si?

Reply:
Veldig likt:

```@prefix br:    <https://raw.githubusercontent.com/Informasjonsforvaltning/organization-catalogue/master/src/main/resources/ontology/organization-catalogue.owl#> .
@prefix nybrreg:     <https://ny.brreg.uri.no/begreper#> .
@prefix adms:  <http://www.w3.org/ns/adms#> .
@prefix dct:   <http://purl.org/dc/terms/> .
@prefix org:   <http://www.w3.org/ns/org#> .
@prefix rov:   <http://www.w3.org/ns/regorg#> .
@prefix skos:  <http://www.w3.org/2004/02/skos/core#> .
@prefix foaf:  <http://xmlns.com/foaf/0.1/> .

<http://localhost:5000/organizations/974760673>
        a                      rov:RegisteredOrganization ;
        br:nace                "84.110" ;
        br:municipality        <http://localhost:5000/administrativeEnheter/kommune/id/172833> ;
        br:norwegianRegistry   <http://localhost:5000/enhetsregisteret/api/enheter/974760673> ;
        br:orgPath             "/STAT/912660680/974760673" ;
        br:sectorCode          "6100" ;
        org:subOrganizationOf  <http://localhost:5000/organizations/912660680> ;
        rov:legalName          "REGISTERENHETEN I BRØNNØYSUND" ;
        dct:identifier         "974760673" ;
        rov:orgType            [ a               skos:Concept ;
                                 skos:prefLabel  "ORGL"
                               ] ;
        rov:orgStatus          nybrreg:NormalAktivitet ;
        rov:registration       [ a                  adms:Identifier ;
                                 dct:issued         "1999-02-03" ;
                                 skos:notation      "974760673" ;
                                 adms:schemaAgency  "Brønnøysundregistrene"
                               ] ;
        foaf:name              "Brønnøysundregistrene"@nb .```

Reply:
Bra. Køyr!

Reply:
Eller: Det er vel @azaxarov som må "resolve" denne tråden.